cirq-core/cirq/experiments/single_qubit_readout_calibration.py

```diff
@@ -12,22 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Dict, Iterable, TYPE_CHECKING
+from typing import Iterable, TYPE_CHECKING
 
-import dataclasses
-import time
+from cirq._compat import deprecated
+from cirq.experiments.parallel_readout_calibration import estimate_parallel_readout_errors
+from cirq.experiments.readout_experiment_result import ReadoutExperimentResult
 
-import numpy as np
-
-from cirq import circuits, ops
 
 if TYPE_CHECKING:
     import cirq
 
 
-@dataclasses.dataclass(frozen=True)
-class SingleQubitReadoutCalibrationResult:
+@deprecated(
+    deadline="v0.13",
+    fix="use cirq.experiments.ReadoutExperimentResult instead",
+    name="cirq.experiments.SingleQubitReadoutCalibrationResult",
+)
+class SingleQubitReadoutCalibrationResult(ReadoutExperimentResult):
```

Comment:
From our discussion offline, we didn't think there was a compelling enough reason to want to rename this class with no major changes to existing properties or functionality. So this is fine to leave for now and maybe update in a future PR.

Reply:
Reverted change.